src/main.cpp

```diff
@@ -1,8 +1,50 @@
 #include <Arduino.h>
 #include <iodefs.h>
-#include <wifi.h>
+#include <WiFi.h>
+#include <MQTT.h>
+#include <secrets.h>
 #include <utils.h>
 
+const char ssid[] = SECRET_SSID;
+const char pass[] = SECRET_PASS;
+const char mqtt_address[] = SECRET_SERVICE;
+
+WiFiClient net;
+MQTTClient client;
+
+unsigned long lastMillis = 0;
+
+void connect()
+{
+    Serial.print("checking wifi...");
+    while (WiFi.status() != WL_CONNECTED)
+    {
+        Serial.print(".");
+        delay(1000);
+    }
+
+    Serial.print("\nconnecting...");
+    while (!client.connect("arduino", "public", "public"))
```

Comment:
while (!client.connect("arduino", "need-for-speed", "Qo7h3F9ZejnTq1uA"))

Reply:
ok